SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -591,7 +591,9 @@ Worker_ComponentId USpatialClassInfoManager::ComputeActorInterestComponentId(con
 		return SpatialConstants::ALWAYS_RELEVANT_COMPONENT_ID;
 	}
 
-	if (GetDefault<USpatialGDKSettings>()->bEnableNetCullDistanceInterest)
+	// Don't add NCD component to player controller and server only actors as we don't want client's to gain interest in them
```

Comment:
Here we utilise the NCD component to provide functionality that read ACLs used to provide, specifically client's only checking out their player controller, and clients not checking out serveronly actor types.